CHANGELOG.md

```diff
@@ -18,6 +18,7 @@ This project adheres to [Semantic Versioning](https://semver.org/).
 
 ### Deprecated
 
+- Deprecate `DeclarationBlock::createBorderShorthand()` (#578)
```

Comment:
We either also need to have the second deprecated method in the changelog, or only deprecate `createBorderShorthand` in this PR.

Reply:
It's fixed now.